### PR TITLE
Qt/FSUI: Translation fixes 

### PR DESCRIPTION
**Pull request by kamfretoz**
### Description of Changes

Addresses:
- `About Window -> The "Version: " string is untranslatable`
- `Game Properties -> Details -> Type -> Type names are untranslatable.`

### Rationale behind Changes

More translation the gooder.

### Suggested Testing Steps

See if i broke anything.

### CODE REVIEW

File: pcsx2/GameList.cpp [hunk 1]
```diff
@@ -99,14 +99,14 @@ const char* GameList::EntryTypeToString(EntryType type)
 
 const char* GameList::EntryTypeToDisplayString(EntryType type)
 {
-	static std::array<const char*, static_cast<int>(EntryType::Count)> names = {{"PS2 Disc", "PS1 Disc", "ELF"}};
+	static std::array<const char*, static_cast<int>(EntryType::Count)> names = {{TRANSLATE("GameList", "PS2 Disc"), TRANSLATE("GameList","PS1 Disc"), TRANSLATE("GameList", "ELF")}};
```
Comment (F0bes) [inline pcsx2/GameList.cpp:102]: Missing a space before "PS1 Disc"

**Review by F0bes — CHANGES_REQUESTED** [review]
Just one formatting issue.

File: pcsx2/GameList.cpp (hunk 1 above)
Comment (kamfretoz) [reply pcsx2/GameList.cpp]: Done

**Review by F0bes — APPROVED** [review]
lgtm

**Pull request merged**